app/views/welcome/index.html.erb

```diff
@@ -1 +1,27 @@
+<header class="masthead">
+  <div class="container h-100">
+    <div class="row h-100 align-items-center">
+      <div class="col-12 text-center">
+        <h1 class="font-weight-light">ObituaryHub</h1>
+        <!-- insert logo image here instead of header -->
+        <br>
+        <p class="lead">a place to celebrate life<br>and honor death</p>
+      </div>
+    </div>
+  </div>
+</header>
 
+<!-- Page Content -->
+<section class="py-5" style= "background-color:white;">
+  <div class="container">
+    <h2 class="font-weight-light">How?</h2>
+On the navigation bar, to the left you will find links to see collections of obituaries based on different criteria. To the right you will find search options. The text field is best used for names, and if you want to search based on other parameters you can select advanced search options.In order to upload a new obituary, you will need to create an account. You can create an account here on our website, or authorize using Facebook. In the future we plan to expand on this functionality so you can import posts from Facebook.
+  </div>
+  <br>
+  <div class="container">
+    <h2 class="font-weight-light">Why?</h2>
+    <p>In my home growing up, my parents would often read the obituary section of the newspaper and discuss people who had passed away at the breakfast table. They would send condolences to friends and family of the deceased if they were acquainted, and sometimes make a plan to attend a funeral or make a donation based on the information contained in the obituary. Fast forward to today. I find out that friends have died from Facebook. Death announcements and public expressions of mourning are interspersed amongst the political posts and advertisements.</p>
+
+    <p>We believe these beautiful expressions of love and sorrow deserve a better ecosystem. This is why we created ObituaryHub. It is built with the dignity of the deceased in mind as well as the tender hearts of those in mourning. It is a place to celebrate life and honor death.</p>
+  </div>
```

Comment:
This is really great Stella! It makes me excited to be a part of this project!

Reply:
love love love